bindata/network/ovn-kubernetes/common/alert-rules.yaml

```diff
@@ -26,3 +26,15 @@ spec:
       for: 20m
       labels:
         severity: warning
+    - alert: NodeWithOVNContollerDisconnectedFromSBDB
+      annotations:
+        summary: All nodes should be having stable connections between their ovn-controller and SBDB, {{"{{"}} $labels.node {{"}}"}} does not.
+        # runbook_url: TODO by https://issues.redhat.com/browse/SDN-3444
+        description: |
+          Networking is degraded on nodes that do not have a healthy ovn-controller <-> SBDB connection. We will fire an alert
+          if ovn-controller to SBDB connection has been down for >= 10 minutes at a stretch.
+      expr: |
+        ovn_controller_southbound_database_connected == 0
```

Comment:
The JIRA card said 10minutes. Do we want to fire when its 5 minutes or 10 minutes? Probably 5 makes sense since having controller not talk to SBDB for 5 minutes is a huge thing. Cool I'll remove the `for`

Reply:
and nice read you linked me, max_over_time makes total sense; learnt some new things..glad I picked up this task